Makefile

```diff
@@ -299,6 +299,16 @@ quickstart: $(KUSTOMIZE) manifests #EXHELP Generate the installation release man
 
 ##@ Docs
 
+.PHONY: crd-ref-docs
+API_REFERENCE_FILENAME := operator-controller-api-reference.md
+crd-ref-docs: $(CRD_REF_DOCS)
+	rm -f $(ROOT_DIR)/docs/drafts/$(API_REFERENCE_FILENAME)
+	$(CRD_REF_DOCS) --source-path=$(ROOT_DIR)/api \
```

Comment:
There are a wealth of options and configuration we can use we we generate these; if there are any that seem interesting or necessary [here](https://github.com/elastic/crd-ref-docs?tab=readme-ov-file#usage) please let me know! Otherwise for now I'm just dumping the entire API reference into one `.md` file. We could for instance build a config file to ignore the `ClusterExtensionList` type if we wanted to.